chain/epoch_manager/src/lib.rs

```diff
@@ -439,7 +439,8 @@ impl EpochManager {
         Ok(epoch_info.validators[seat.tenant as usize].clone())
     }
 
-    pub fn get_all_block_producer_info(
+    /// Returns all unique block producers in current epoch sorted by validator_stake, with indicator on whether they are slashed or not.
```

Comment:
It is sorted by account id

Reply:
I checked `proposals_to_epoch_info()`, you're absolutely right.